src/test/java/seedu/address/testutil/ProjectsFolderBuilder.java

```diff
@@ -0,0 +1,38 @@
+package seedu.address.testutil;
+
+import seedu.address.model.ProjectsFolder;
+import seedu.address.model.project.Project;
+
+/**
+ * A utility class to help with building ProjectsFolder objects.
+ * Example usage: <br>
+ *     {@code ProjectsFolder pf = new ProjectsFolderBuilder().withProject(project).build();}
+ */
+public class ProjectsFolderBuilder {
+
+    private ProjectsFolder projectsFolder;
+
+    public ProjectsFolderBuilder() {
```

Comment:
this 2 method missing javadoc also, as for other builder, you put javadoc for constructor also